prow/config.yaml

```diff
@@ -150,6 +150,42 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  GoogleCloudPlatform/k8s-multicluster-ingress:
+  - name: pull-kubernetes-multicluster-ingress-test
+    agent: kubernetes
+    context: pull-kubernetes-multicluster-ingress-test
+    branches:
+    - master
+    rerun_command: "/test pull-kubernetes-multicluster-ingress-test"
+    trigger: "(?m)^/test( all| pull-kubernetes-multicluster-ingress-test),?(\\s+|$)"
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:0.4
+        args:
+        - "--repo=github.com/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
this needs to be `"--repo=github.com/GoogleCloudPlatform/$(REPO_NAME)=$(PULL_REFS)"`

Reply:
oh right, good catch